package.json

```diff
@@ -48,6 +48,7 @@
     "concurrently": "^3.6.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.5.0",
+    "enzyme-matchers": "^6.0.4",
```

Comment:
Huh - it looks like this is used but it's confusing that Enzyme's API obscures this...

Reply:
Yeah, I wasn't sure how to set this up.  I couldn't find any docs but it seems to be working after installing it.  I should try uninstalling it to see if the `toExist()` matcher still works.